withTime.js

```diff
@@ -0,0 +1,24 @@
+const EE = require('./eventEmitter');
```

Comment:
let's give it a bit clearer name `EventEmitter`

Reply:
done